include/zephyr/devicetree.h

```diff
@@ -4172,6 +4230,15 @@
 #define DT_INST_NODE_HAS_PROP_AND_OR(inst, prop) \
 	DT_INST_NODE_HAS_PROP(inst, prop) ||
 
+/**
+ * @brief Macro to add ULL postfix to the devicetree address constants
+ */
```

Comment:
```suggestion
/**
 * @def DT_U64_C
 * @brief Macro to add ULL postfix to the devicetree address constants
 */
```

because comment is not attached to the macro

Reply:
Thanks, improved